packages/veto-backend/src/api/cyphernode/bumpFee.ts

```diff
@@ -0,0 +1,16 @@
+import { AxiosInstance, AxiosResponse } from 'axios'
+import cyphernodeClient from './client'
+
+const cyphernode: AxiosInstance = cyphernodeClient()
+
+type Args = {
+  txid: string
+  confTarget: number
+}
+
+export default function spend({ txid, confTarget }: Args): Promise<AxiosResponse> {
+  return cyphernode.post('/bumpfee', {
+    txid,
+    confTarget: Number(confTarget),
```

Comment:
Since we are using Typescript to validate our types, we can do type conversion as soon as we get the data. This will  prevent us from doing type conversion everywhere.